kore/src/main/scala/org/kframework/attributes/Att.scala

```diff
@@ -170,96 +196,96 @@ object Att {
+  final val COOL = Key("cool", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val DEPENDS = Key("depends", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val ELEMENT = Key("element", KeyType.BuiltIn, KeyParameter.Required)
+  final val EXIT = Key("exit", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FORMAT = Key("format", KeyType.BuiltIn, KeyParameter.Required)
+  final val FRESH_GENERATOR = Key("freshGenerator", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FUNCTION = Key("function", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FUNCTIONAL = Key("functional", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val GROUP = Key("group", KeyType.BuiltIn, KeyParameter.Required)
+  final val HASKELL = Key("haskell", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val HEAT = Key("heat", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val HYBRID = Key("hybrid", KeyType.BuiltIn, KeyParameter.Optional)
+  final val HOOK = Key("hook", KeyType.BuiltIn, KeyParameter.Required)
+  final val IDEM = Key("idem", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val IMPURE = Key("impure", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INDEX = Key("index", KeyType.BuiltIn, KeyParameter.Required)
+  final val INITIAL = Key("initial", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INITIALIZER = Key("initializer", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INJECTIVE = Key("injective", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INTERNAL = Key("internal", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val KAST = Key("kast", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val KLABEL = Key("klabel", KeyType.BuiltIn, KeyParameter.Required)
+  final val KORE = Key("kore", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val LABEL = Key("label", KeyType.BuiltIn, KeyParameter.Required)
+  final val LATEX = Key("latex", KeyType.BuiltIn, KeyParameter.Required)
+  final val LEFT = Key("left", KeyType.BuiltIn, KeyParameter.Optional)
+  final val LOCATIONS = Key("locations", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val MACRO = Key("macro", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val MACRO_REC = Key("macro-rec", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val MAINCELL = Key("maincell", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val MEMO = Key("memo", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val ML_BINDER = Key("mlBinder", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val ML_OP = Key("mlOp", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val MULTIPLICITY = Key("multiplicity", KeyType.BuiltIn, KeyParameter.Required)
+  final val NO_EVALUATORS = Key("no-evaluators", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val NON_ASSOC = Key("non-assoc", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val NON_EXECUTABLE = Key("non-executable", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val NOT_LR1 = Key("not-lr1", KeyType.BuiltIn, KeyParameter.Optional)
```

Comment:
I thought this one was forbidden?

Reply:
And RuleGrammarGenerator is generating this attribute with some values as well

https://github.com/runtimeverification/k/blob/a863a1e4618067b7027a12bcdbd2d1fca3ba8273/kernel/src/main/java/org/kframework/parser/inner/RuleGrammarGenerator.java#L535-L537

Reply:
Introduced by #1583